orderer/consensus/etcdraft/node.go

```diff
@@ -171,6 +171,11 @@ func (n *node) run(campaign bool) {
 			// to the followers and them writing to their disks. Check 10.2.1 in thesis
 			n.send(rd.Messages)
 
+		case <-n.storage.WALSyncC:
+			if err := n.storage.Sync(); err != nil {
+				n.logger.Debugf("Failed to sync raft log, error: %s", err)
```

Comment:
why debug? we would want to know about it if we failed to sync

Reply:
Yes, We can make it warn level.